recipe/meta.yaml

```diff
@@ -73,10 +73,12 @@ test:
     - jinja2
     - cffi
     - scipy
+    # At least NumPy 1.26.0 has an issue with arctanh on glibc<=2.17.
+    # ref: https://github.com/conda-forge/numba-feedstock/pull/128#issuecomment-1795031396
+    - numpy !=1.26.0  # [linux and x86_64]
     - ipython
     - setuptools
     - tbb
-    - llvm-openmp              # [osx]
```

Comment:
Good catch. We should already be getting this as a dependency of `numba` from `run_exports/strong` of `llvm-openmp` in `requirements/build`. So this shouldn't actually be needed. Probably leftover from before

Reply:
Yep.
And in case that "run-exported" dependency is not desired as a hard dependency (like TBB), we should add an ignore for that.
For consistency, I instead added `libgomp` for Linux, though.

Reply:
Yeah this is a fairly old recipe (preceding the compiler migration 😳). Recall also there was some special history with LLVM OpenMP on macOS. So likely leaving out GNU OpenMP was an oversight. Thanks for fixing that as well! 🙏